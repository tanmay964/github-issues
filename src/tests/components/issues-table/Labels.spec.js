import React from 'react';
import renderer from 'react-test-renderer';
import Labels from '../../../components/issues-table/Labels';

it('renders correctly', () => {
  const labels = [{
    id: 123,
    color: 'red',
    name: 'Test Label 1',
  },
  {
    id: 132,
    color: 'green',
    name: 'Test Label 2',
  },
  ];
  const tree = renderer
    .create(<Labels labels={labels} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
