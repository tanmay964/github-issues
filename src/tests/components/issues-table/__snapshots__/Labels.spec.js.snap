// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="sc-bwzfXH fBfiXH"
>
  <span
    className="sc-bdVaJa esexkF"
  >
    Test Label 1
  </span>
  <span
    className="sc-bdVaJa fhALxq"
  >
    Test Label 2
  </span>
</span>
`;
